src/sb_utils/src/RvizUtils.cpp

```diff
@@ -6,20 +6,40 @@
  */
 #include <RvizUtils.h>
 
-using namespace std;
 using namespace visualization_msgs;
-using namespace snowbots;
-
-Marker RvizUtils::createMarker(vector<geometry_msgs::Point> points,
-                               Marker::_color_type color,
-                               Marker::_scale_type scale,
-                               string frame_id,
-                               string ns,
-                               int type,
-                               int id) {
+// using namespace snowbots;
+
+namespace snowbots {
+namespace RvizUtils {
+    /**
+     * Helper function that sets up common marker parameters
+     *
+     * @param scale the scale
+     * @param frame_id the frame id
+     * @param ns the namespace
+     * @param type the type of marker
```

Comment:
this is just a number, could we specify that it's from `visualization_msgs::Marker::*`? Same goes if we pass the type like this in other places here.